### DISCUSSION TITLE
<input> type = "number" accpets strings on FireFox!!

### DISCUSSION THREAD

**Discussion by AnilSeervi** [mdn/mdn-community#598] [Content] [answered]
The `<input>` element of type `number` seems to accpets strings into the field on FireFox while other chromium based browsers doesn't.
Even on MDN Docs of [`<input type="number">`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/number), on the first line, it says :
> They include built-in validation to reject non-numerical entries.

And on [specs](https://html.spec.whatwg.org/multipage/input.html#number-state-(type=number)) it says : 
> User agents must not allow the user to set the value to a non-empty string that is not a valid floating-point number.

Shoudn't this behaviour be same across all the browser.

**Comment by promet99** [comment, marked as answer]
This is a persisting problem. You can find 5, 6 years old stack overflow questions discussing this matter.

This is Firefox browser's problem, so it should be discussed there, I guess

**Comment by pepelsbey** [comment]
Closing as answered/settled.
